cdap-ui/app/cdap/components/PipelineTriggers/index.js

```diff
@@ -48,6 +52,17 @@ export default class PipelineTriggers extends Component {
       let state = PipelineTriggersStore.getState().triggers;
       if (state.enabledTriggers.length === this.state.enabledTriggersCount) { return; }
 
+
+      // This is to set the default tab. During initial construction, the store
+      // has not been updated with data, therefore the count is not accurate.
+      if (!updateOnce) {
+        this.setState({
+          enabledTriggersCount: state.enabledTriggers.length,
+          activeTab: state.enabledTriggers.length === 0 ? 1 : 0
+        });
+        return;
```

Comment:
Don't we need to set `updateOnce = true` so that further changes to `PipelineTriggersStore` don't trigger this block of code?